tests/visual/percy-storybook/common/index.tsx

```diff
@@ -0,0 +1,2 @@
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
Import all the bootstrap things. This is applied to all tests (whether they all import this file or not). As currently set up, this means that we are no longer testing a "vanilla" version of the table in these tests and we would be required to duplicate this test run to do so (e.g. an additional ~1600 snapshots per build)

There are some differences between vanilla and the bootstrap styled table, as far as I can tell the main difference seems to be a slightly different font size. Nothing that strikes as compromising the vanilla test cases.